CefSharp/Handler/IAccessibilityHandler.cs

```diff
@@ -0,0 +1,25 @@
+﻿// Copyright © 2010-2018 The CefSharp Authors. All rights reserved.
+//
+// Use of this source code is governed by a BSD-style license that can be found in the LICENSE file.
+
+namespace CefSharp
+{
+    /// <summary>
+    /// Implement this interface to receive accessibility notification when accessibility events have been registered. 
+    /// The methods of this class will be called on the UI thread.
```

Comment:
Change to `It's important to note this event is fired on a CEF UI thread, which by default is not the same as your application UI thread`